Source/Meadow.Foundation.Peripherals/Displays.Ch1115/Samples/Ch1115_Sample/MeadowApp.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Meadow;
+using Meadow.Devices;
+using Meadow.Foundation.Displays;
+using Meadow.Foundation.Graphics;
+using System.Threading.Tasks;
+
+namespace Displays.Ch1115_Sample
+{
+    public class MeadowApp : App<F7FeatherV2>
+    {
+        //<!=SNIP=>
+
+        MicroGraphics graphics;
+
+        public override Task Initialize()
+        {
+            Resolver.Log.Info("Initializing...");
+
+            var ch1115 = new Ch1115
+            (
+                spiBus: Device.CreateSpiBus(),
+                chipSelectPin: Device.Pins.D02,
+                dcPin: Device.Pins.D01,
+                resetPin: Device.Pins.D00,
+                width: 128,
+                height: 64
+            );
+
+            graphics = new MicroGraphics(ch1115);
+            graphics.CurrentFont = new Font8x8();
+
+            return base.Initialize();
+        }
+
+        public override Task Run()
+        {
+            graphics.Clear();
+            graphics.DrawTriangle(10, 10, 50, 50, 10, 50, Meadow.Foundation.Color.Red);
+            graphics.DrawRectangle(20, 15, 40, 20, Meadow.Foundation.Color.Yellow, true);
```

Comment:
if its Monochrome, especifying colors are not needed, right?

Reply:
uuhh ... correct ... I'll fix ;)